lib/internal/crypto/webcrypto.js

```diff
@@ -238,7 +251,7 @@ async function deriveKey(
       throw lazyDOMException('Unrecognized name.');
   }
 
-  return importKey('raw', bits, derivedKeyAlgorithm, extractable, keyUsages);
+  return this.importKey('raw', bits, derivedKeyAlgorithm, extractable, keyUsages);
```

Comment:
```suggestion
  return importKey('raw', bits, derivedKeyAlgorithm, extractable, keyUsages);
```

Actually, I've tested with Firefox, Chromium, Safari, and Deno: it looks like it's not looking up the  `importKey` function property but rather use the original function directly. Can you add a test case for this and this other methods as well I assume):
```js
crypto.subtle.importKey(
    'raw',
    crypto.getRandomValues(new Uint8Array(4)),
    'PBKDF2',
    false,
    ['deriveKey']).then(key => {
        subtle.importKey = common.mustNotCall();
        return crypto.subtle.deriveKey({
            name: 'PBKDF2',
            hash: 'SHA-512',
            salt: crypto.getRandomValues(new Uint8Array()),
            iterations: 5
          }, key, {
            name: 'AES-GCM',
            length: 256
          }, true, ['encrypt', 'decrypt']);
    }).then(common.mustCall());
```

Reply:
Not sure I follow @aduh95. I either need to use subtle.importKey or this.importKey, or importKey.call(this, ...). Either way the result will be the same, no?